docs/release-notes/experiment-deadlock.rst

```diff
@@ -0,0 +1,6 @@
+:orphan:
+
+**Bug Fixes**
+
+-  Kubernetes: Fix a bug where reading an experiment's resource pool name would hang indefinitely
```

Comment:
I don't think it Kubernetes specific and I think it came on after move job so I would try and relate the release note to what customers would experience

Maybe something like

```
API: Fix an issue where calling ``det job update`` could cause jobs to be prevented from scheduling and ``det job ls`` to hang. 
```